internal/controllers/ipaddressclaim.go

```diff
@@ -19,25 +19,18 @@ package controllers
 import (
 	"context"
 	"fmt"
+	"slices"
```

Comment:
the failures here are happening because probably you also need to bump it into github actions like https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/blob/main/.github/workflows/pullrequests.yaml#L17 and https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/blob/main/.github/workflows/pullrequests.yaml#L30 and https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/blob/main/.github/workflows/pullrequests.yaml#L41